LootLockerServerSDK/Source/LootLockerServerSDK/Private/LootLockerServerHttpClient.cpp

```diff
@@ -43,6 +51,7 @@ void ULootLockerServerHttpClient::SendRequest_Internal(HTTPRequest InRequest) co
 
 	Request->SetHeader(TEXT("User-Agent"), UserAgent);
 	Request->SetHeader(TEXT("User-Instance-Identifier"), UserInstanceIdentifier);
+	Request->SetHeader(TEXT("SDK-Version"), SDKVersion);
```

Comment:
Should we add something more to this than just "SDKversion"? Since it says the same thing as the SDK, omething could be sent from the Server-SDK and we would only know by the version number.